### ISSUE TITLE
[Documentation] inconsistent and duplicated fields in api

### ISSUE THREAD

**Issue by Tails128** [thepracticaldev/dev.to#9065]
**Describe the bug**

Currently `tags` and `tag_list` have two different behavior:
```
tags: string;
tag_list: string[];
```
and:
```
tags: string;
tag_list: string[];
```

It would be nice to have it unified, in order not to get confused with it... but only having one of the two versions (preferably the string[] one) would be even better so that the server has to send fewer data and that the user has to download fewer.

**and**
The same (choosing one of the two fields) would go for `published_timestamp` and `published_at`, which seem to be the same field, but duplicated.
[edit] also the field `published` (which is already absent in some of the API) could be removed and be deducted such that the final user can deduce it as: `bool published = published_at != undefined`.

**Comment by jacobherrington** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @Tails128, some are known issues, can you add the others to https://github.com/thepracticaldev/dev.to/issues/4637 ?

I'm going to close this over the previous ticket. Changing them it's going to unfortunately break the API v0. 

They'll definitely get fixed in v1.
